mrtarget/common/LookupHelpers.py

```diff
@@ -150,7 +147,6 @@ def _get_mp(self):
         obj = OntologyClassReader()
         mp_uri = Config.ONTOLOGY_CONFIG.get('uris', 'mp')
         opentargets_ontologyutils.mp.load_mammalian_phenotype_ontology(obj, mp_uri)
-        obj.rdf_graph = None
```

Comment:
Also here - I think this line is important in saving memory by free objects that should no longer be used for garbage collection.

Reply:
Done!